src/partials/header.html

```diff
@@ -0,0 +1,39 @@
+<div class="header">
+    <div class="container">
+        <img class="header__logo"
+            src="../images/logo 1.jpg"
+            alt="Logo IceCream"
+            width="199"
+            height="50"/>
+        <div class="header__hamburger">
+            <svg class="header__hamburger--icon">
+                <use href="../images/icons.svg$icon-hamburger"></use>
+            </svg>
+            <svg class="header__hamburger--icon">
+                <use href="../images/icons.svg$icon-hamburger"></use>
+            </svg>
+            <svg class="header__hamburger--icon">
+                <use href="../images/icons.svg$icon-hamburger"></use>
+            </svg>
```

Comment:
Jeśli dobrze rozumiem, że tu jest hamburger 3 razy bo to są 3 razy po jednej kresce to umęczysz się pozycjonując to potem. Dużo prościej przy stylizowaniu potem byłoby zaznaczyć w figmie całą ikonę i wstawić po prostu jako jeden element.